__tests__/app.test.js

```diff
@@ -152,3 +152,58 @@ describe("* GET/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("6. GET/api/articles/:article_id/comments", () => {
+  test("returns status code 200", () => {
+    return request(app).get("/api/articles/3/comments").expect(200);
+  });
+
+  test("responds with a single matching article", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              article_id: expect.any(Number),
+              body: expect.any(String),
+              author: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
+
+  test("returns comments in date order newist first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
+  test("returns status code 400 when given non-existent article_id", () => {
+    return request(app)
+      .get("/api/articles/5345/comments")
+      .expect(404)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe("not found");
+      });
+  });
+
+  test("returns status code 400 when given invalid article_id", () => {
+    return request(app)
+      .get("/api/articles/zzNdf/comments")
+      .expect(400)
+      .then((response) => {
+        const msg = response.body.msg;
+        expect(msg).toBe("bad request");
+      });
+  });
+});
```

Comment:
You are missing a 200 test for a valid article that has no comments, it should return an empty array